packages/ketcher-react/src/script/editor/shared/closest.ts

```diff
@@ -406,10 +441,10 @@ function findClosestRgroupAttachmentPoints(
       };
 }
 
-function findClosestRxnArrow(restruct, pos) {
-  let minDist = null;
-  let refPoint = null;
-  let ret = null;
+function findClosestRxnArrow(restruct: ReStruct, pos: Vec2) {
+  let minDist = Number.POSITIVE_INFINITY;
```

Comment:
Also here, can it potentially affect some existing functionality?

Reply:
Same as above